dapr/proto/runtime/v1/dapr.proto

```diff
@@ -72,6 +72,13 @@ service Dapr {
   // InvokeActor calls a method on an actor.
   rpc InvokeActor (InvokeActorRequest) returns (InvokeActorResponse) {}
 
+  // GetConfiguration gets configuration from configuration store.
+  rpc GetConfiguration(GetConfigurationRequest) returns (GetConfigurationResponse) {}
+
+  // SubscribeConfiguration subscribes the update events of configuration items changed.
+  // The event will be sent by OnConfigurationEvent() in app callback
+  rpc SubscribeConfiguration(SubscribeConfigurationRequest) returns (google.protobuf.Empty) {}
```

Comment:
I like this because it is an example of push API (instead of using callback API). Please, add a way to do this via CRD too.